### DISCUSSION TITLE
The vk-io-callback module cannot be found

### DISCUSSION THREAD

**Discussion by vendetttta** [negezor/vk-io#543] [Q&A]
Hi. I am writing a bot on node js, I want to run it, but when downloading the module it gives an error, how can this be solved?

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/vk-io-callback - Not found
npm ERR! 404
npm ERR! 404  'vk-io-callback@latest' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:

**Comment by nitreojs** [comment]
provide your code please
